code/__DEFINES/achievements.dm

```diff
@@ -42,6 +43,7 @@
 #define BOSS_MEDAL_KINGGOAT   "King Goat Killer"
 
 #define BOSS_MEDAL_MINER_CRUSHER	  	"Blood-drunk Miner Crusher"
+#define BOSS_MEDAL_FROSTMINER_CRUSHER	"Demonic-frost Miner Crusher"
```

Comment:
keep the whitespace formatting and indent this like the rest

Reply:
doesn't look like this when I look on my local so don't know what to say